pkg/gossiper/gossiper.go

```diff
@@ -18,13 +20,13 @@ const HEARTBEAT_PATH = "heartbeat"
 const HEARTBEAT_TIMEOUT = 1000
 
 type gossiper struct {
```

Comment:
any reason we want to expose its content but not the struct? 

If you expose the content for retrieving content, then there should be a getter which also dereference these content.